pandas/tests/groupby/test_groupby.py

```diff
@@ -2709,6 +2709,24 @@ def test_single_element_list_grouping():
     assert result == expected
 
 
+@pytest.mark.parametrize(
+    "level_arg, multiindex", [([0], False), ((0,), False), ([0], True), ((0,), True)]
+)
```

Comment:
This is a small enough case that it is perfectly fine as-is, but wanted to mention that you can stack parameterizations and it will take the cartesian product:

```
@pytest.mark.parametrize("level_arg", [[0], (0,)])
@pytest.mark.parametrize("multiindex", [False, True])
```